src/openassetio-python/bridge/python/hostApi.cpp

```diff
@@ -24,8 +24,9 @@ ManagerImplementationFactoryInterfacePtr createPluginSystemManagerImplementation
     LoggerInterfacePtr logger) {  // NOLINT(performance-unnecessary-value-param)
   // Get Python class.
   py::object pyClass =
-      py::module_::import("openassetio.pluginSystem.PluginSystemManagerImplementationFactory")
-          .attr("PluginSystemManagerImplementationFactory");
+      py::module_::import(
+          "openassetio.pluginSystem.PythonPluginSystemManagerImplementationFactory")
```

Comment:
Will do. I wasn't sure about this, since it's in the `python` namespace so prefixing with `Python` seemed redundant.  But better to be super explicit for the avoidance of any doubt.

Reply:
Yeah - I was just imagining that when we have a unifying plugin system, we may well still want to expose that to python too, which then might cause ambiguity, so having it match the class name is probably quite valuable.

Reply:
Renamed in dcd5528b